man/meson.build

```diff
@@ -6,9 +6,9 @@ subdir('rules')
 want_man = get_option('man')
```

Comment:
You might want to remove this and `want_html` below, since they're redefined based on `xsltproc`.

nitpick below, perhaps `xsltproc.found() and not get_option('man').disabled()` might be more readable? ("and not" maybe better than "not ... and ...")